src/components/modules/RecentAndTrending/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { PureComponent } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ModuleBox, OverlapScaffold, Post, Pattern } from '@/components/modules';
+import { Container, Row, Image } from '@/components/primitives';
+import { Responsive } from '@/components/utils';
+import { BREAKPOINTS, CONTAINER_PADDING, COLOR_MAP } from '@/constants';
+import { TagTileBoxModuleData, PostType, OFFSET_DIRECTION, GridOrder } from '@/types';
+import PostGrid  from './PostGrid';
+import TrendingLine from '@/assets/images/trending-line.svg';
+
+interface Props extends TagTileBoxModuleData {
+    postsOne: PostType[];
```

Comment:
Let's call these recentPosts and trendingPosts just to make it clear

Reply:
Renamed these props.